### ISSUE TITLE
Private class fields support

### ISSUE THREAD

**Issue by tmcw** [observablehq/parser#64]
Same kind of issue as bigints and async generators: [Shipping Chrome (2 versions ago)](https://developers.google.com/web/updates/2018/12/class-fields) includes new syntax:

```
class Example {
  classFields = 0;
  #privateField = 0;
  static #field = 4;
  static method() { };
}
```

This should possibly start from [this class fields plugin](https://github.com/acornjs/acorn-class-fields).

**Comment by esperanc** [comment]
This should definitely be ranked top-priority.

**Comment by mbostock** [comment]
Fixed in #205.
